### ISSUE TITLE
Ramsey/uuid 3.x

### ISSUE THREAD

**Issue by mnwk** [prooph/event-store-symfony-bundle#23]
It is still not possible to use ramsey uuid 3.x with symfony.
```
Problem 1
    - Installation request for prooph/event-store-symfony-bundle ^0.2.2 -> satisfiable by prooph/event-store-symfony-bundle[v0.2.2].
    - Conclusion: remove ramsey/uuid 3.7.1
    - Conclusion: don't install ramsey/uuid 3.7.1
    - prooph/event-store-symfony-bundle v0.2.2 requires prooph/event-store ^6.3 -> satisfiable by prooph/event-store[6.x-dev, v6.3.0, v6.4.0, v6.5.0, v6.5.1, v6.6.0].
    - prooph/event-store 6.x-dev requires prooph/common ~3.7 -> satisfiable by prooph/common[3.x-dev, v3.7, v3.7.1].
    - prooph/event-store v6.3.0 requires prooph/common ~3.7 -> satisfiable by prooph/common[3.x-dev, v3.7, v3.7.1].
    - prooph/event-store v6.4.0 requires prooph/common ~3.7 -> satisfiable by prooph/common[3.x-dev, v3.7, v3.7.1].
    - prooph/event-store v6.5.0 requires prooph/common ~3.7 -> satisfiable by prooph/common[3.x-dev, v3.7, v3.7.1].
    - prooph/event-store v6.5.1 requires prooph/common ~3.7 -> satisfiable by prooph/common[3.x-dev, v3.7, v3.7.1].
    - prooph/event-store v6.6.0 requires prooph/common ~3.7 -> satisfiable by prooph/common[3.x-dev, v3.7, v3.7.1].
    - prooph/common 3.x-dev requires ramsey/uuid ~2.8 -> satisfiable by ramsey/uuid[2.8.0, 2.8.1, 2.8.2, 2.8.3, 2.8.4, 2.9.0, 2.x-dev].
    - prooph/common v3.7 requires ramsey/uuid ~2.8 -> satisfiable by ramsey/uuid[2.8.0, 2.8.1, 2.8.2, 2.8.3, 2.8.4, 2.9.0, 2.x-dev].
    - prooph/common v3.7.1 requires ramsey/uuid ~2.8 -> satisfiable by ramsey/uuid[2.8.0, 2.8.1, 2.8.2, 2.8.3, 2.8.4, 2.9.0, 2.x-dev].
    - Can only install one of: ramsey/uuid[2.8.0, 3.7.1].
    - Can only install one of: ramsey/uuid[2.8.1, 3.7.1].
    - Can only install one of: ramsey/uuid[2.8.2, 3.7.1].
    - Can only install one of: ramsey/uuid[2.8.3, 3.7.1].
    - Can only install one of: ramsey/uuid[2.8.4, 3.7.1].
    - Can only install one of: ramsey/uuid[2.9.0, 3.7.1].
    - Can only install one of: ramsey/uuid[2.x-dev, 3.7.1].
    - Installation request for ramsey/uuid (locked at 3.7.1) -> satisfiable by ramsey/uuid[3.7.1].
```

**Comment by codeliner** [comment]
Current master branch is up-to-date but we don't have a new release, because tests and documentation is missing. You could use dev master for now. API of the bundle should not change. Check https://github.com/prooph/proophessor-do-symfony to see how the new bundle can be used within a symfony 4 application.

**Comment by enumag** [comment]
This can be closed now - fixed by v0.3.
